### DISCUSSION TITLE
Implementing LIONESS on netZooPy

### DISCUSSION THREAD

**Discussion by Enakshi-Saha** [netZoo/netZooPy#250] [General]
Hi, I am trying to generate LIONESS networks for 538 male samples (GTEx lung tissue and TCGA LUAD combined) using netZooPy. 
In the function lioness there is an argument "save_single". If  "save_single = True", the function save LIONESS networks for individual samples in separate files (format specified by the user) right after generating that network. If "save_single = False", then after generating all the networks, they are arranged as columns in a single matrix and the entire matrix is saved into a file, separate files for individual samples are not created. 
Now here's the issue I'm having: I load the expression, PPI and Motif files into matrices and then pass these matrices into "panda". Then pass the PANDA output into lioness with the argument "save_single = False". Here's my code for generating LIONESS networks:
`panda_obj = Panda(expression_data, motif_data, ppi_data,computing='gpu',precision='single',save_tmp=False,save_memory = False, remove_missing=True, keep_expression_matrix = True,modeProcess = 'intersection', with_header = True)
lioness_obj = Lioness(panda_obj,computing='gpu', save_dir='lioness_output', save_single = False, export_filename='Lung_lioness_female_TMM.txt')`

All the networks are generated as expected, however while creating the matrix with individual samples as columns, the program generates a ValueError: Shape of passed values is (14632320, 538), indices imply (538, 538). The output is not stored. 
What should I do?

**Comment by violafanfani** [comment]
Ehi @Enakshi-Saha I tried to replicate your error, but I do not get the same error with the test data I have. 
Would you mind adding/double check the format of the data you are passing? Also, do you get the same error with 'cpu'? It might be worth checking with just a few samples (start = 1, end = 3).
